ocis-phoenix/CHANGELOG.md

```diff
@@ -0,0 +1,373 @@
+# Changelog for [unreleased] (UNRELEASED)
+
+The following sections list the changes in ocis-phoenix unreleased.
+
+[unreleased]: https://github.com/owncloud/ocis/ocis-phoenix/compare/v0.13.0...master
+
+## Summary
+
+* Bugfix - Fix external app URLs: [#218](https://github.com/owncloud/product/issues/218)
+* Change - Remove pdf-viewer from default apps: [#85](https://github.com/owncloud/ocis/ocis-phoenix/pull/85)
+
+## Details
+
+* Bugfix - Fix external app URLs: [#218](https://github.com/owncloud/product/issues/218)
+
+   The URLs for the default set of external apps was hardcoded to localhost:9200. We fixed that by
+   using relative paths instead.
+
+   https://github.com/owncloud/product/issues/218
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/83
+
+* Change - Remove pdf-viewer from default apps: [#85](https://github.com/owncloud/ocis/ocis-phoenix/pull/85)
+
+   OCIS-web doesn't have a dedicated pdf-viewer app anymore but instead uses the default browser
+   behaviour for PDFs. We removed pdf-viewer from the set of default apps.
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/85
+
+# Changelog for [0.13.0] (2020-08-25)
+
+The following sections list the changes in ocis-phoenix 0.13.0.
+
+[0.13.0]: https://github.com/owncloud/ocis/ocis-phoenix/compare/v0.12.0...v0.13.0
+
+## Summary
+
+* Change - Update Phoenix: [#81](https://github.com/owncloud/ocis/ocis-phoenix/pull/81)
+
+## Details
+
+* Change - Update Phoenix: [#81](https://github.com/owncloud/ocis/ocis-phoenix/pull/81)
+
+   Updated phoenix from v0.15.0 to v0.16.0
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/81
+   https://github.com/owncloud/phoenix/releases/tag/v0.16.0
+
+# Changelog for [0.12.0] (2020-08-19)
+
+The following sections list the changes in ocis-phoenix 0.12.0.
+
+[0.12.0]: https://github.com/owncloud/ocis/ocis-phoenix/compare/v0.11.0...v0.12.0
+
+## Summary
+
+* Change - Enable Settings and Accounts apps by default: [#80](https://github.com/owncloud/ocis/ocis-phoenix/pull/80)
+* Change - Update Phoenix: [#79](https://github.com/owncloud/ocis/ocis-phoenix/pull/79)
+
+## Details
+
+* Change - Enable Settings and Accounts apps by default: [#80](https://github.com/owncloud/ocis/ocis-phoenix/pull/80)
+
+   The default ocis-web config now adds the frontend of ocis-accounts and ocis-settings to the
+   builtin web config.
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/80
+
+* Change - Update Phoenix: [#79](https://github.com/owncloud/ocis/ocis-phoenix/pull/79)
+
+   Updated phoenix from v0.14.0 to v0.15.0
+
+   https://github.com/owncloud/ocis/ocis-phoenix/pull/79
+   https://github.com/owncloud/phoenix/releases/tag/v0.15.0
+
+# Changelog for [0.11.0] (2020-08-17)
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)